src/olympia/reviewers/views.py

```diff
@@ -1386,6 +1400,27 @@ def retrieve(self, request, *args, **kwargs):
         )
         return Response(serializer.data)
 
+    @action(
+        detail=True,
+        methods=['get', 'put', 'delete'],
+        permission_classes=ReviewAddonVersionMixin.permission_classes)
```

Comment:
This would let add-on authors delete the `DraftComment` of a reviewer?  That seems... not good opsec.

Reply:
uhm, fair point, I'll limit this. Actually, fair point, should this actually be limited to the actual author of the draft comment? e.g, if two reviewers are reviewing the same version, should they produce their own draft comment instances? 

cc @wagnerand for some ideas about this too

Reply:
> if two reviewers are reviewing the same version, should they produce their own draft comment instances?

I'd say yes.

Reply:
Alrighty, implemented in fc8d0f1 and 80159dc